py-polars/polars/show_versions.py

```diff
@@ -69,3 +70,36 @@ def _get_dep_version(dep_name: str) -> str:
 
     # all our dependencies (as of 2022-08-11) implement __version__
     return module_version
+
+
+def _closure_() -> _typing_.Callable[[], _typing_.Any]:
```

Comment:
The method name could be more descriptive. I propose to call this `build_info`, and import this method directly in `__init__.py`

Reply:
The function is called `build_info`. Closure `_closure_` is here to create a private scope and initiate private variables that `build_info()` returns. Closure executed once and then deleted. This is standard way to overcome python's limitation of not having proper private or static variables.

Reply:
`_closure_()` is gone now. See the updated PR.